<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QComboBox" name="comBBook">
       <item>
        <property name="text">
         <string>All book</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Text:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pBFind">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Found in the files:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="comBChapter">
       <item>
        <property name="text">
         <string>All chapters</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QTableWidget" name="tableFiles">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <column>
        <property name="text">
         <string>Module</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Book</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Chapter</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type module</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="LAFilesFoundLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comBModule">
       <item>
        <property name="text">
         <string>All modules</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="LEFind">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
